docs/src/pages/components/data-grid/export/export.md

```diff
@@ -6,23 +6,40 @@ title: Data Grid - Export
 
 <p class="description">Easily export the rows in various file formats such as CSV, Excel, or PDF.</p>
 
-## CSV export
+## Enabling export
+
+### Default Toolbar
 
-The easiest way to enable the CSV export is to pass the `GridToolbar` component in the `Toolbar` [component slot](/components/data-grid/components/#toolbar).
+To enable the export selector, pass the `GridToolbar` component in the `Toolbar` [component slot](/components/data-grid/components/#toolbar).
 
 {{"demo": "pages/components/data-grid/export/ExportDefaultToolbar.js", "bg": "inline"}}
 
-### Custom toolbar
+### Custom Toolbar
+
+The export selector is provided in a stand alone component named `GridToolbarExport`. You can use it in a custom toolbar component as follow.
```

Comment:
Could we avoid talking about "selector" for other things than the state selectors ?
I know that we are doing it on a few places already. But I feel like it can create confusion.

Reply:
Completely agree
What do you think about "export menu"?